packages/core-ethereum/src/indexer/index.ts

```diff
@@ -226,10 +228,21 @@ class Indexer extends EventEmitter {
           continue
         }
       }
+      confirmedEvents.push(event)
```

Comment:
This effectively stores a reverse copy of `this.unconfirmedEvents` in `confirmedEvents`. Could be beneficial to change

```
const event = this.unconfirmedEvents.pop()
```
to
```
const event = this.unconfirmedEvents.unshift()
```

Reply:
we are sorting the list in the next step so this is unimportant.